<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pear129</class>
 <widget class="QMainWindow" name="Pear129">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layout" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
</string>
    </property>
    <widget class="QLabel" name="mainLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>821</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;맑은 고딕&quot;;</string>
     </property>
     <property name="text">
      <string>메뉴를 선택하세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tableButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>490</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(255, 5, 38, 100), stop:1 rgb(255, 85, 127));</string>
     </property>
     <property name="text">
      <string>조회하기(Table)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="graphButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>490</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(255, 164, 16, 100), stop:1 rgb(255, 153, 10));</string>
     </property>
     <property name="text">
      <string>조회하기(Graph)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recommendButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>490</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(120, 255, 154,100), stop:1 rgb(85, 255, 127));</string>
     </property>
     <property name="text">
      <string>창업 추천받기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mapButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>490</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(63, 127, 191, 100), stop:1 rgb(0, 85, 255));</string>
     </property>
     <property name="text">
      <string>지도에서 보기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>490</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(170, 85, 255, 100), stop:1  rgb(173, 58, 255));</string>
     </property>
     <property name="text">
      <string>종료하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wcButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>490</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
background-color:qlineargradient(spread:pad, x0:1, y0:1, x0:1, y1:1, stop:0 rgba(85, 170, 255, 100), stop:1 rgb(170, 255, 255));</string>
     </property>
     <property name="text">
      <string>키워드 모아보기</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>700</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <zorder>widget</zorder>
    <zorder>mainLabel</zorder>
    <zorder>tableButton</zorder>
    <zorder>graphButton</zorder>
    <zorder>recommendButton</zorder>
    <zorder>mapButton</zorder>
    <zorder>exitButton</zorder>
    <zorder>wcButton</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
